src/spyglass/common/common_region.py

```diff
@@ -52,6 +56,65 @@ def fetch_add(
         )
         query = BrainRegion & key
```

Comment:
Consider removing this function or only allowing brain regions from a specific atlas.